source/braille.py

```diff
@@ -1427,6 +1425,41 @@ def setDisplayByName(self, name, isFallback=False):
 			self.setDisplayByName("noBraille", isFallback=True)
 			return False
 
+	def _getTablePath(self, table):
+		if not os.path.isabs(table):
+			# This is a stock table.
+			return os.path.join(TABLES_DIR, table)
+		return table
+
+	def _getTableList(self, table):
+		return [self._getTablePath(table), PATTERNS_TABLE]

```

Comment:
This worries me because of liblouis silliness. From the liblouis documentation about the table list (square brackets used for emphasis):

> liblouis knows where to find all the tables that have been distributed with it. So you can just give a table name such as en-us-g2.ctb and liblouis will load it. You can also give a table name which includes a path. [If this is the first table in a list, all the tables in the list must be on the same path.] You can specify a path on which liblouis will look for table names by setting the environment variable LOUIS_TABLEPATH. This environment variable can contain one or more paths separated by commas. On receiving a table name liblouis first checks to see if it can be found on any of these paths. If not, it then checks to see if it can be found in the current directory, or, [if the first (or only) name in a table list, if it contains a path name, can be found on that path]. If not, it checks to see if it can be found on the path where the distributed tables have been installed.

According to this, it will ignore the path specified for any tables other than the first. So, if the first table is within an add-on, that means braille-patterns.cti won't be found. The stuff about the environment variable and distribution tables doesn't apply; I'm pretty sure you can't change the environment for a running process, only for subsequent processes, and we don't want to compile in an absolute path for distribution tables.

All of this said, if this is indeed the case, surely you would have seen this problem when testing? Maybe the documentation is actually wrong?